readthedocs/projects/static-src/projects/js/import.js

```diff
@@ -236,7 +245,22 @@ function ProjectImportView (instance, config) {
         if (current_id == id) {
             id = null;
         }
+        // This needs to use deferred updates
         self.filter_org(id);
+        self.filter_own(false);
```

Comment:
I think we can get around the need for deferred updates here if we avoid resetting both the `filter_org` and `filter_own`. i think this means we instead have a singular `filter_by()` observable. This would remove the duplicated updates to the observable performing the ajax request.

Reply:
I'm not too convinced of having just one list for the filters yet.

We can't remove the deferred updates because we need to modify the `page_current` also.